homeassistant/helpers/config_validation.py

```diff
@@ -446,6 +447,19 @@ def socket_timeout(value):
             raise vol.Invalid('Invalid socket timeout: {err}'.format(err=_))
 
 
+def isip(value: Any) -> str:
```

Comment:
Please remove this and instead use `from ipaddress import ip_address` and use `ip_address` as a validator.